src/command/info/activity.ts

```diff
@@ -0,0 +1,88 @@
+import { Collection, Message, StageChannel, VoiceChannel } from "discord.js";
+import { MinehutCommand } from "../../structure/command/minehutCommand";
+import fetch from 'node-fetch';
+
+export default class ActivityCommand extends MinehutCommand {
+    constructor() {
+        super('activity', {
+			aliases: ['activity'],
+			cooldown: 10000,
+            ratelimit: 1,
+			description: {
+				content: 'To start an activity',
+				usage: '<activity>',
+				examples: ['youtube'],
+			},
+			category: 'info',
+			args: [
+				{
+					id: 'activityname',
+					type: 'string',
+					prompt: {
+						start: (msg: Message) =>
+							`${msg.author}, what type of activity would you like to start?\n\`youtube\`, \`fishington\`, \`chess\`, \`checkers\`, \`betrayal\`, \`doodlecrew\`, \`wordsnacks\`, \`lettertile\`, \`poker\`, \`spellcast\``,
+						retry: (msg: Message) =>
+							`${msg.author}, please provide an activity name.`,
+					},
+				}
+			],
+		});
+	}
+
+    async exec(msg: Message, { activityname }: { activityname: string}) {
+
+        if(!msg.member?.voice.channel) {
+            await msg.channel.send(`${msg.author}, please join a voice channel.`);
+            return;
+        }
+
+        const vc: VoiceChannel | StageChannel = msg.member.voice.channel;
+
+        if(!msg.member.permissions.has(1n << 39n)) { //START_EMBEDDED_ACTIVITIES
+			await msg.channel.send(`${process.env.EMOJI_CROSS} You do not have permission to start activities in <#${vc.id}>.`);
+            return;
+        }
+
+		const applications: Collection<string, string> = new Collection();
+		applications.set("youtube", "880218394199220334");
```

Comment:
The use of quotes for string literal declarations is inconsistent with the codebase. As jellz said, however, running Prettier with the configuration would fix this inconsistency.